keyboards/whtkbrd/readme.md

```diff
@@ -0,0 +1,15 @@
+# Whtkbrd
+
+![Whtkbrd](https://i.imgur.com/xQbjgjS.jpeg)
```

Comment:
This uses a smaller, but still high res image that works better on github markdown pages
```suggestion
![Whtkbrd](https://i.imgur.com/xQbjgjSh.jpeg)
```

Reply:
Good to know! Applied this suggestion.